tests/integration/pay.test.js

```diff
@@ -178,12 +180,12 @@ describe('PAY ROUTES - POST /pay', () => {
 });
 
 describe('PAY ROUTES - GET /hours-balance-details', () => {
```

Comment:
Dans le service est-ce que tu pourrais remplace le this par exports ?

Reply:
done